.env.example

```
@@ -34,3 +34,11 @@ OPENROUTER_MODEL=google/gemini-2.5-flash
 OPENROUTER_MAX_COMPLETION_TOKENS=1200
 OPENROUTER_TIMEOUT_MS=30000
 GOOGLE_OAUTH_ALLOW_INSECURE_TLS="false" # Set to "true" only for local/dev testing with self-signed TLS certs
+
+# Stripe Payment Configuration
+STRIPE_SECRET_KEY=sk_test_your_stripe_secret_key
+STRIPE_WEBHOOK_SECRET=whsec_your_stripe_webhook_secret
+# Optional: fallback Stripe checkout currency when invoice currency is unsupported by your Stripe account
+STRIPE_FALLBACK_CURRENCY=usd
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The STRIPE_FALLBACK_CURRENCY key should go before the STRIPE_SECRET_KEY key

(UnorderedKey)